<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popupPage</class>
 <widget class="QDialog" name="popupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_placedNext">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已放，下一个</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_jumpThisOption">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>跳过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_backPreviousOption">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>试剂名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>试剂容量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>摆放位置：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_showReagentName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_showReagentVolume">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_showReagentlocation">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
